.github/workflows/opensearch-sink-opensearch-integration-tests.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       matrix:
         java: [11]
-        opensearch: [1.0.1, 1.1.0, 1.2.4, 1.3.2, 2.0.0]
+        opensearch: [1.0.1, 1.1.0, 1.2.4, 1.3.2, 2.0.1, 2.1.0, 2.2.1]
```

Comment:
We updated our OpenSearch-rest-high-level-client to use 1.3.5, we should add that here.
2.3.0 is released yesterday. should we also include that here?

Reply:
Yes, we should do that.

Reply:
@asifsmohammed , I updated the tests to use 1.3.5 instead of 1.3.2, and I added 2.3.0.